app/controllers/structures_controller.rb

```diff
@@ -51,4 +51,10 @@ def display_error(messages)
   def decrypted_token
     Argo.verifier.verified(params[:item_id], purpose: :view_token)
```

Comment:
I wonder if the path above, `item_user_version_structure_path(@document.id, @view_token)`, will work with this line unchanged. This path evaluates to `/items/druid:bb000zn0114/user_versions/skret-t0k3n/structure`, which means in the controller `params[:item_id]` is `druid:...` and `params[:user_version_id]` is the token, so I don't understand how `Argo.verified.verified(params[:item_id], purpose: :view_token)` is verifying a non-token.

Reply:
We changed this.